test/opt/test_pass_manager.cpp

```diff
@@ -54,6 +56,17 @@ TEST(PassManager, Interface) {
   EXPECT_STREQ("strip-debug", manager.GetPass(0)->name());
   EXPECT_STREQ("null", manager.GetPass(1)->name());
   EXPECT_STREQ("strip-debug", manager.GetPass(2)->name());
+
+  manager.AddPass<opt::NullPass>(1u);
```

Comment:
Instead of using `NullPass`, I would suggest that we create another pass here with various constructors for testing purpose only. `NullPass` is a user-facing pass, it should have meaningful interface.